app/cli/internal/policydevel/lint.go

```diff
@@ -96,15 +96,25 @@ func Lookup(absPath, config string, format bool) (*PolicyToLint, error) {
 	}
 
 	if fileInfo.IsDir() {
-		if err := scanDirectory(policy, absPath); err != nil {
-			return nil, err
+		// If it's a directory, look for policy.yaml
```

Comment:
I'd remove the option of passing a dir. You always pass a file path with a default

Reply:
what I mean that it looks strange that the default is a directory when we could make it a file path relative to cwd?

I believe we might have some similar code in the CLI that handles this case, default with relative path or explicit absolute path. Golang might have a helper for that too.